src/panels/lovelace/hui-root.ts

```diff
@@ -675,7 +706,10 @@ class HUIRoot extends LitElement {
     const configBackground = viewConfig.background || this.config.background;
 
     if (configBackground) {
-      view.style.setProperty("--lovelace-background", configBackground);
+      root.parentElement!.style.setProperty(
```

Comment:
This is dangerous, if the dom structure changes this will break. Change the query selectors to the decorator style and cache them. You can then just query the `#layout` element.

Als is setting the background for unused entities line 677-681 still needed?

Reply:
I'll drop all extra styling for now like balloob suggests below, but I was thinking this too, since unused entities changed and is now in its own list box there's really no need for it either.